rust-toolchain.toml

```diff
@@ -1,2 +1,6 @@
 [toolchain]
 channel = "{{ toolchain }}"
+{%- if toolchain == "nightly" %}
```

Comment:
maybe it's worth to define a dedicated variable to decide this?

like when we decide to use a specific nightly or once Xtensa support gets into mainline Rust and we use nightly for Xtensa, too

Reply:
What about deleting the `toolchain`? We only use it here (and we already need an if statement) and in the Dockerfile where it will only add an if

Reply:
I guess we should keep the `toolchain` variable as long as we have `esp` and `nightly`. I meant just have an additional bool variable for this one `if`

Reply:
Why not use the [arch variable](https://github.com/esp-rs/esp-template/blob/main/pre-script.rhai#L3) then?

Reply:
Oh yes - `arch` would be better

Reply:
Updated!